CHANGELOG.md

```diff
@@ -5,6 +5,116 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+* `nan` to indicate `no_channel`, split dim singleton (#6090)
+* Compatible padding mode (#6076)
+* Allow for missing `filename_or_obj` key (#5980)
+* `Spacing` pixdim in-place change (#5950)
+* Add warning in `RandHistogramShift` (#5877)
+* Exclude `cuCIM` wrappers from `get_transform_backends` (#5838)
+#### data
+* `__format__` implementation of MetaTensor (#6523)
+* `channel_dim` in `TiffFileWSIReader` and `CuCIMWSIReader` (#6514)
+* Prepend `"meta"` to `MetaTensor.__repr__` and `MetaTensor.__str__` for easier identification (#6214)
+* MetaTensor slicing issue (#5845)
+* Default writer flags (#6147)
+* `WSIReader` defaults and tensor conversion (#6058)
+* Remove redundant array copy for WSITiffFileReader (#6089)
+* Fix unused arg in `SlidingPatchWSIDataset` (#6047)
+* `reverse_indexing` for PILReader (#6008)
+* Use `np.linalg` for the small affine inverse (#5967)
+#### metrics and losses
+* Removing L2-norm in contrastive loss (L2-norm already present in CosSim) (#6550)
+* Fixes the SSIM metric (#6250)
+* Efficiency issues of Dice metrics (#6412)
+* Generalized Dice issue (#5929)
+* Unify output tensor devices for multiple metrics (#5924)
+#### networks
+* Make `RetinaNet` throw errors for NaN only when training (#6479)
+* Improves NVFuser import check (#6399)
+* Add `device` in `HoVerNetNuclearTypePostProcessing` and `HoVerNetInstanceMapPostProcessing` (#6333)
+* Enhance hovernet load pretrained function (#6269)
+* Access to the `att_mat` in self-attention modules (#6493)
+* Optional swinunetr-v2 (#6203)
+* Add transform to handle empty box as training data for `retinanet_detector` (#6170)
+* GPU utilization of DiNTS network (#6050)
+* A pixelshuffle upsample shape mismatch problem (#5982)
+* GEGLU activation function for the MLP Block (#5856)
+* Constructors for `DenseNet` derived classes (#5846)
+* Flexible interpolation modes in `regunet` (#5807)
+#### bundle
+* Optimized the `deepcopy` logic in `ConfigParser` (#6464)
+* Improve check and error message of bundle run (#6400)
+* Warn or raise ValueError on duplicated key in json/yaml config (#6252)
+* Default metadata and logging values for bundle run (#6072)
+* `pprint` head and tail in bundle script (#5969)
+* Config parsing issue for substring reference (#5932)
+* Fix instantiate for object instantiation with attribute `path` (#5866)
+* Fix `_get_latest_bundle_version` issue on Windows (#5787)
+#### engines and handlers
+* MLflow handler run bug (#6446)
+* `monai.engine` training attribute check (#6132)
+* Update StatsHandler logging message (#6051)
+* Added callable options for `iteration_log` and `epoch_log` in TensorBoard and MLFlow (#5976)
+* `CheckpointSaver` logging error (#6026)
+* Callable options for `iteration_log` and `epoch_log` in StatsHandler (#5965)
+#### misc
+* Avoid creating cufile.log when `import monai` (#6106)
+* `monai._extensions` module compatibility with rocm (#6161)
+* Remove warning message: deprecated tensor.storage usage (#6105)
```

Comment:
Should move this one to the "Removed"?

Thanks.

Reply:
that's fixing the bug described here https://github.com/Project-MONAI/MONAI/issues/6104, I've updated the description.